src/git.js

```diff
@@ -242,19 +242,49 @@ class Git {
 		const tree = []
 		for (const treeObject of output.split('\n')) {
 			const [ mode, type, sha ] = treeObject.split(/\s/)
-			const treeObjectPath = treeObject.split('\t')[1]
+			const file = treeObject.split('\t')[1]
 
 			const treeEntry = {
 				mode,
 				type,
-				content: await this.getBlobContent(sha),
```

Comment:
We don't upload the content of the file when creating the tree, the files/blobs are references by its SHA only.